### ISSUE TITLE
XenPlatform features

### ISSUE THREAD

**Issue by deantheiceman** [xendit/xendit-php#105]
How do we handle sub accounts feature and creating sub accounts?

**Comment by stanleynguyen** [comment]
Hi @deantheiceman, unfortunately XenPlatform features are not available in our PHP SDK yet but rest assured it's in our roadmap. For now, you have to stick with manually making HTTP requests

**Comment by deantheiceman** [comment]
@stanleynguyen could we use https://github.com/xendit/xendit-php#use-custom-http-client to inject the header temporarily? but can't seem to get the custom http client working

eg.

```
class HttpCliImp implements HttpClientInterface {
    private $_guzz;

    public function __construct(Guzzle $guzz)
    {
        $this->_guzz = $guzz;
    }

    public function request($method, $uri, array $options = [])
    {
        return $this->_guzz->request($method, $uri, $options);
    }
}

class XenditClient
{
    public function __construct($team)
    {

        $client = new Guzzle(
            [
                'headers' => [
                    'for-user-id' => 123,
                ],
          ]
        );

        Xendit::setHttpClient(new HttpCliImp($client));
  }
}
```

But I'm getting this error:

```
local.ERROR: Call to a member function getBody() on null {"exception":"[object] (Error(code: 0): Call to a member function getBody() on null at /var/www/html/vendor/xendit/xendit-php/src/HttpClient/GuzzleClient.php:133)
```

**Comment by David-Kurniawan** [comment]
Hopefully can release my PR ASAP #142 😃 

@stanleynguyen

**Comment by xen-HendryZheng** [comment]
xenPlatform has been released on [#142](https://github.com/xendit/xendit-php/pull/142)
Assisting to close this issue.
